test/src/tests-core-sdk.js

```diff
@@ -229,30 +219,11 @@ describe('core SDK', function() {
         done();
     })
 
-    it('should set client id', function(done) {
```

Comment:
Not sure what CGID was ever for, but the v3 events endpoint does not have this so I removed this test